src/routes/jira/atlassian-connect/jira-atlassian-connect-get.ts

```diff
@@ -160,6 +167,16 @@ const modules = {
 
 export const moduleUrls = compact(map([...modules.adminPages, ...modules.generalPages], "url"));
 
+// Remove this function when CREATE_BRANCH flag is complete
+const moduleModifier = async (modules) => {
```

Comment:
Instead of adding and then deleting depending upon the condition, can't we just add conditionally? 
What if we just do the FF check before, if true then we add the `actions` property in the `modules`, and if not keep the `modules` as it is.

Reply:
Yeah, I agree with @krazziekay

Reply:
Instead of adding and then deleting depending upon the condition, can't we just add conditionally?
What if we just do the FF check before, if true then we add the actions property in the modules, and if not keep the modules as it is.

Reply:
Yeah, I also agree

Reply:
Updated to be additive